packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -133,8 +133,10 @@ void testWidgets(
 /// If the callback is asynchronous, make sure you `await` the call
 /// to [benchmarkWidgets], otherwise it won't run!
 ///
-/// Benchmarks must not be run in checked mode. To avoid this, this
-/// function will print a big message if it is run in checked mode.
+/// Benchmarks must not be run in checked mode, because the performance is not
+/// representative. To avoid this, this function will print a big message if it
+/// is run in checked mode. Unit tests of this method pass [mayRunWithAsserts],
```

Comment:
nit: Since mayRunWithAsserts is just a method argument and not a property in the surrounding context it should be enclosed with ` instead of [].

Reply:
Done.